athena-federation-integ-test/pom.xml

```diff
@@ -0,0 +1,187 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>aws-athena-query-federation</artifactId>
+        <groupId>com.amazonaws</groupId>
+        <version>1.0</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>athena-federation-integ-test</artifactId>
+    <packaging>jar</packaging>
+    <name>Amazon Athena Query Federation Integ Test</name>
+
+    <properties>
+        <slf4jVersion>1.7.30</slf4jVersion>
+        <log4j2Version>2.13.3</log4j2Version>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.amazonaws</groupId>
+            <artifactId>aws-athena-federation-sdk</artifactId>
+            <version>2020.46.1</version>
+            <exclusions>
+                <!-- replaced with jcl-over-slf4j -->
+                <exclusion>
+                    <groupId>commons-logging</groupId>
+                    <artifactId>commons-logging</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
```

Comment:
Why does this need the federation SDK? I thought this integration test only targets Athena?

Reply:
Removed Federation SDK dependency and replaced with Athena and Guava dependencies.